TSB genertaed lib from scoped package not AoT compatible

When we generate a lib for scoped package using tsb. The metadata.json changes flatModuleID to safe-package name rather than actual package-name. This cause AoT to not find the module.
```
ERROR in ./apps/app/src/app/app.module.ngfactory.js
Module not found: Error: Can’t resolve ‘mycompany-auth’ in ‘/Users/system/projects/ngseed/apps/app/src/app’
 @ ./apps/app/src/app/app.module.ngfactory.js 23:10-32
 @ ./apps/app/src/main.ts
 @ multi webpack-dev-server/client?http://0.0.0.0:0 ./apps/app/src/main.ts
```
However when we updated tsconfig.json to pass flatModuleID same as package-name then error disappears
```
  "angularCompilerOptions" : {
    "flatModuleId": "@mycompany/auth"
  }
```

Here is snippet of metadata.json (before change)
```
{
.....
 "origins": {
  //entries of file
  },
  "importAs": "mycompany-auth"
}
```
Here is snippet of metadata.json after changes
```
{
.....
 "origins": {
  //entries of file
  },
  "importAs": "@mycompany/auth"
}
```
I think we need a change here 
https://github.com/aminpaks/typescript-library-bundler/blob/master/src/config-helpers.ts#L71
We should pass raw module id to https://github.com/aminpaks/typescript-library-bundler/blob/master/src/config-helpers.ts#L47
and convert to safe-id for usage at this place
https://github.com/aminpaks/typescript-library-bundler/blob/master/src/config-helpers.ts#L60

I can create a PR for the change if required

--- a/src/config-helpers.ts
+++ b/src/config-helpers.ts
@@ -44,7 +44,7 @@ export function getSafePackageName(name: string): string | undefined {
   return name.replace(/[^\w-]/g, '-').replace(/^-+/, '');
 }
 
-export function createNGCConfig(filePath: string, moduleId: string, configs?: TSConfigs): void {
+export function createNGCConfig(filePath: string, moduleId: string, configs?: TSConfigs, flatModuleId?:string): void {
   const result = mergeInto(<TSConfigs>{
     compilerOptions: {
       outDir: './ngc-compiled',
@@ -68,7 +68,7 @@ export function createNGCConfig(filePath: string, moduleId: string, configs?: TS
     result.angularCompilerOptions!.skipTemplateCodegen = true;
   }
   if (isNil(result.angularCompilerOptions!.flatModuleId)) {
-    result.angularCompilerOptions!.flatModuleId = moduleId;
+    result.angularCompilerOptions!.flatModuleId = flatModuleId || moduleId;
   }
   if (!isNil(result.include)) {
     delete result.include;

--- a/src/main.ts
+++ b/src/main.ts
@@ -115,7 +115,7 @@ export async function main(projectPath: string, configFilePath?: string, buildOp
   // AngularCompiler configurations
   const ngcBuildDir = path.resolve(buildDir, 'ngc-compiled');
   const ngcConfigPath = path.resolve(buildDir, 'tsconfig-ngc.json');
-  createNGCConfig(ngcConfigPath, moduleId, configs);
+  createNGCConfig(ngcConfigPath, moduleId, configs, packageName);
 
   // Compile with NGC
   await ngcCompiler(ngcConfigPath, { basePath: buildDir });

--- a/test/configs.spec.ts
+++ b/test/configs.spec.ts
@@ -28,6 +28,39 @@ describe('ConfigHelpers', () => {
     assert.property(defaultTSConfig, 'compilerOptions');
 
     const moduleId = 'check-mod';
+    const flatModuleId = '@scope/module';
+    createNGCConfig(ngcConfigsPath, moduleId, defaultTSConfig, flatModuleId);
+
+    ({ configs: defaultTSConfig, error  } = parseConfigFile(ngcConfigsPath));
+
+    assert.isDefined(defaultTSConfig, 'files');
+    assert.isArray(defaultTSConfig.files);
+    assert.deepEqual(defaultTSConfig.files, ['./src/public_api.ts']);
+
+    assert.isUndefined(error);
+    assert.property(defaultTSConfig, 'compilerOptions');
+    assert.propertyVal(defaultTSConfig.compilerOptions, 'outDir', './ngc-compiled');
+    assert.propertyVal(defaultTSConfig.compilerOptions, 'module', 'es2015');
+
+    assert.property(defaultTSConfig, 'angularCompilerOptions');
+    assert.propertyVal(defaultTSConfig.angularCompilerOptions, 'flatModuleId', flatModuleId);
+    assert.propertyVal(defaultTSConfig.angularCompilerOptions, 'annotateForClosureCompiler', false);
+  });
+
+
+  it('createNGCConfig: no flatModuleId', () => {
+
+    const ngcConfigsPath = path.resolve(__dirname, 'tsconfig.ngc.json');
+
+    let { configs: defaultTSConfig, error } = parseConfigFile(defaultTSConfigPath);
+    defaultTSConfig.angularCompilerOptions = {
+      annotateForClosureCompiler: false,
+    };
+
+    assert.isUndefined(error);
+    assert.property(defaultTSConfig, 'compilerOptions');
+
+    const moduleId = 'check-mod';
     createNGCConfig(ngcConfigsPath, moduleId, defaultTSConfig);
 
     ({ configs: defaultTSConfig, error  } = parseConfigFile(ngcConfigsPath));

